src/redux/stocks/stock.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable max-len */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Reply:
I has been corrected in the recent commit that I made 👍